test/extended/crdvalidation/apiserver.go

```diff
@@ -0,0 +1,45 @@
+package crdvalidation
+
+import (
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	exutil "github.com/openshift/origin/test/extended/util"
+)
+
+var _ = g.Describe("[Suite:openshift/crdvalidation/apiserver] APIServer CR fields validation", func() {
+	var (
+		oc              = exutil.NewCLI("cluster-basic-auth", exutil.KubeConfigPath())
+		apiServerClient = oc.AdminConfigClient().ConfigV1().APIServers()
+	)
+	defer g.GinkgoRecover()
+
+	g.It("additionalCORSAllowedOrigins", func() {
+		apiServerOrig, err := apiServerClient.Get("cluster", metav1.GetOptions{})
+		o.Expect(err).NotTo(o.HaveOccurred())
+		defer func() {
+			apiServerClient.Update(apiServerOrig)
```

Comment:
if one of the invalid regex is not caught, this update will not succeed because the resource version is increased. I would remove the whole defer here.